asmcli/lib/installation_dependencies.sh

```diff
@@ -43,7 +43,8 @@ required_iam_roles() {
 }
 # [END required_iam_roles]
 
-old_required_apis() {
+# [START required_apis]
+required_apis() {
```

Comment:
is there a reason for the function name change? I think this was changed in #1152 for a reason, but I am not sure.

Reply:
I didn't change this, it was required to run `scripts/release-asm/precommit` for the PR

Reply:
the precommit only merges the changes made by you so maybe you changed this by accident or have a stale commit. I would suggest keeping only the changes made by you and rebase on HEAD.